marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/jobInfo/jobInfo.api

```diff
@@ -0,0 +1,13 @@
+{
+    "functionName": "latestJobInfo",
+    "params": [
+        {
+            "name": "model",
```

Comment:
I recommend calling this parameter "entityCollection" since it is assumed to be a collection value.

Reply:
Included in https://github.com/marklogic/marklogic-data-hub/pull/3011/commits/e1d00a2bc5e0cc4d4afe9e00cfd0fc146f4f53aa